src/components/Training/Formats.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import breakpoint from 'styled-components-breakpoint';
+
+import { SectionTitle, Subtitle, BodyPrimary } from '../Typography';
+import { Grid, Row, Col } from '../grid';
+import BlueBackground from '../Common/BlueBackground';
+import CustomisedBulletpoint from '../Common/CustomisedBulletpoint';
+
+const TitleRow = styled(Row)`
+  padding-top: ${({ theme }) => theme.space[4]};
+  padding-bottom: ${({ theme }) => theme.space[4]};
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+    padding-bottom: ${({ theme }) => theme.space[6]}
+  `}
+`;
+
+const FormatsRow = styled(Row)`
+  padding-bottom: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('tablet')`
+    padding-bottom: ${({ theme }) => theme.space[7]}
+  `};
+`;
+
+const FormatCol = styled(Col)`
+  padding-bottom: ${({ theme }) => theme.space[4]};
+
+  ${breakpoint('desktop')`
+    padding-bottom: ${({ theme }) => theme.space[0]}
```

Comment:
Use breakpoint ranges like you've done [here](https://github.com/yldio/yld.io/pull/427/files#diff-02ae2f34568c66a2ade52d49b0ffaf7dR36)